### DISCUSSION TITLE
remote sound does not mute after connected

### DISCUSSION THREAD

**Discussion by johnroyer** [rustdesk/rustdesk#8865] [General]
### Bug Description

I expected that RustDest remote  sound will be redirect to RustDesk client, then only client play sound.

After I connected to remote computer and play a video, both client and remote plays sound.

----

client:
- Ubuntu 22.04 (UbuntuMate)
- RustDesk 1.2.7

remote:
- Windows 10
- RustDesk 1.2.7

### How to Reproduce

1. connect to remote computer
2. open and play video
3. sounds comes out from both client and remote

### Expected Behavior

- Sounds plays only on client. 
- Remote computer should be muted or become silence.

### Operating system(s) on local side and remote side

Ubuntu 22.04 on local,  Windows 10 on remote

### RustDesk Version(s) on local side and remote side

both v1.2.7

### Screenshots

I expect settings on this page. For example, "mute remote computer after connected"

![2024-07-28-16:36:50-001](https://github.com/user-attachments/assets/78eb1db8-a005-4d19-a8db-f1077fa20b18)

**Comment by johnroyer** [comment]
any ideas? Thanks

**Comment by xuzhong998** [comment]
I really need this feature too. Video privacy mode alone is not enough; audio privacy mode is also needed.

**Reply by xuzhong998** [reply]
Oh, I found out that we just need to lower the remote volume to zero and make sure to increase the client volume so you can hear it to achieve this

**Reply by johnroyer** [reply]
yes. But it still scare other people if it play sounds suddenly.
